src/System.Management.Automation/engine/ComInterop/ComInvokeBinder.cs

```diff
@@ -114,10 +114,7 @@ private ParameterExpression ParamVariantsVariable
         {
             get
             {
-                if (_paramVariants == null)
-                {
-                    _paramVariants = Expression.Variable(VariantArray.GetStructType(_args.Length), "paramVariants");
-                }
+                _paramVariants ??= Expression.Variable(VariantArray.GetStructType(_args.Length), "paramVariants");
                 return _paramVariants;
             }
         }
```

Comment:
This was similarly modified in .NET Runtime repo but they [went a step further in simplifying it](https://github.com/dotnet/runtime/blob/e1764aa3dd676f16c1d7de28fd752c8ca4e74219/src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComInvokeBinder.cs#L113):
```c#
        private ParameterExpression ParamVariantsVariable => _paramVariants ??= Expression.Variable(VariantArray.GetStructType(_args.Length), "paramVariants");
```

Reply:
Would prefer to keep only null-coalescing assignment in this PR and any other changes in a different PR if needed.